ucm2/Allwinner/A64/PinePhone/HiFi.conf

```diff
@@ -0,0 +1,144 @@
+SectionVerb {
+	EnableSequence [
+		cset "name='Headphone Playback Switch' off"
+		cset "name='Headphone Source Playback Route' DAC"
+		cset "name='Line In Playback Switch' off"
+		cset "name='Line Out Playback Switch' off"
+		cset "name='Line Out Source Playback Route' Mono Differential"
+		cset "name='Mic1 Playback Switch' off"
+		cset "name='Mic2 Playback Switch' off"
+		cset "name='AIF1 DA0 Playback Volume' 160"
+		cset "name='AIF3 ADC Source Capture Route' None"
+		cset "name='AIF2 DAC Source Playback Route' None"
```

Comment:
"None" is not a valid value for this control. When not using AIF3, this should be set to "AIF2". See here: https://github.com/torvalds/linux/blob/master/sound/soc/sunxi/sun8i-codec.c#L844-L855